test/index.test.js

```diff
@@ -0,0 +1,5 @@
+const getArgv = require('src/index.js');
+
+test('obtiene el argv del directorio o archivo', () => {
+  expect(getArgv).toBe(Object);
+});
```

Comment:
Recuerda utilizar la exportación de modulos  
```suggestion
});
module.exports={
//all functions 
}
```